Transmission.xcodeproj/project.pbxproj

```diff
@@ -2947,9 +2951,14 @@
 			isa = XCBuildConfiguration;
 			buildSettings = {
 				CLANG_ENABLE_OBJC_ARC = NO;
+				CLANG_WARN_UNREACHABLE_CODE = NO;
+				GCC_WARN_UNUSED_FUNCTION = NO;
```

Comment:
What's the deal with these warnings? If they don't fail the build I'd rather not to suppress them. If they're coming from 3rd-party code it would be better to report them upstream instead. If they're coming from our code, they need to be fixed (in a separate PR).

Reply:
They are comming from lib-event code, as I remember.
I agree with your statement, I'll revert this part and find if we can do anything about it.

Reply:
Removed these warnings supression.